packages/nitro-rpc-client/scripts/client-runner.ts

```diff
@@ -123,6 +123,7 @@ yargs(hideBin(process.argv))
   .strict()
   .parse();
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 function getRandomElement(col: any[]) {
   return col[Math.floor(Math.random() * col.length)];
 }
```

Comment:
Probably a smell that I needed to fix this. I would guess the `site` package has stricter tsconfig than the `nitro-rpc-client` package. Now we are building the former with `-b` flag, we get a lint error here...